static/loader/empty-scenes/mappings.json

```diff
@@ -256,7 +328,15 @@
     },
     {
       "file": "models/EP_10.glb",
-      "hash": "QmTsvnJp8R2yrdADusiMXHdHjNe7PDr4SaPRd9Y6S1zLku"
+      "hash": "QmXmas210"
```

Comment:
what, this doesn't look good? that's no hash

Reply:
I also wonder. Those scenes are already compiled